IonicPortals/src/main/kotlin/io/ionic/portals/AssetMap.kt

```diff
@@ -0,0 +1,36 @@
+package io.ionic.portals
+
+class AssetMap {
+    val name: String
+    val virtualPath: String
+
+    val path: String
+
+    var prefix: String = "/assets"
```

Comment:
`prefix` here is dead code and not referenced anywhere else

Reply:
Thanks